odxtools/loadfile.py

```diff
@@ -12,19 +14,24 @@ def load_pdx_file(pdx_file: str | Path) -> Database:
     return db
 
 
-def load_odx_d_file(odx_d_file_name: str | Path) -> Database:
+def load_xml_file(xml_file_name: str | Path) -> Database:
```

Comment:
The index file in a PDX is XML, but we don't support loading it

Reply:
yes, but `Database.add_odx_tree()` would be weird IMO, (also, in principle any file can be an "ODX" file if it is an auxiliary file.)